examples/collect_offline_data.py

```diff
@@ -26,7 +26,7 @@
 env_name = 'SafetyAntVelocity-v1'
 size = 1_000_000
 agents = [
-    ('PATH_TO_AGENT', 'epoch-500.pt', 1_000_000),
+    ('train/PPOLag-{SafetyAntVelocity-v1}/seed-000-2024-01-07-21-14-30', 'epoch-500.pt', 1_000_000),
```

Comment:
Should we change it to more general form like 'PATH_TO_AGENT', and make 'train/PPOLag-{SafetyAntVelocity-v1}/seed-000-2024-01-07-21-14-30' as an example in docs?